From e482a26797de3bf65a49a961c014c7b114215f7a Mon Sep 17 00:00:00 2001
From: Shengjiu Wang <shengjiu.wang@freescale.com>
Date: Thu, 19 May 2016 18:24:06 +0800
Subject: [PATCH 1069/5242] MLK-12836: ARM: dts: support display in audio dts
 on imx6ull

commit  6fdf9951e3db2430e072b7be2899a782904edbfd from
https://source.codeaurora.org/external/imx/linux-imx.git

support display in audio dts imx6ull-14x14-ddr3-arm2-cs42888.dts

Signed-off-by: Shengjiu Wang <shengjiu.wang@freescale.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../boot/dts/imx6ull-14x14-ddr3-arm2-cs42888.dts   |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/imx6ull-14x14-ddr3-arm2-cs42888.dts b/arch/arm/boot/dts/imx6ull-14x14-ddr3-arm2-cs42888.dts
index 5130398..b357282 100644
--- a/arch/arm/boot/dts/imx6ull-14x14-ddr3-arm2-cs42888.dts
+++ b/arch/arm/boot/dts/imx6ull-14x14-ddr3-arm2-cs42888.dts
@@ -6,7 +6,7 @@
  * published by the Free Software Foundation.
  */
 
-#include "imx6ull-14x14-ddr3-arm2.dts"
+#include "imx6ull-14x14-ddr3-arm2-lcdif.dts"
 
 / {
 	clocks {
-- 
1.7.9.5

